README.rst

```diff
@@ -166,7 +194,15 @@ In case you are using CWL, Yadage or Snakemake workflow specifications:
 - `reana.yaml using Yadage <reana-yadage.yaml>`_
 - `reana.yaml using Snakemake <reana-snakemake.yaml>`_
 
-We can now install the REANA command-line client, run the analysis and download the resulting file:
+Run the analysis
+----------------
+
+After preparing files, we can run the analysis in two ways:
+
+1. using REANA command line client;
+2. using REANA workflow launcher <docs link here>.
```

Comment:
`<docs link here>` should be replaced... I think you can change the content of README to look like RECAST... i.e. greatly simplify the run-via-CLI part.

Reply:
Actually, it needs to be removed. I missed that.

Reply:
I want to make it very similar to how badges in other demos were done.